### PR TITLE
add missing build dependency on pkg-config to VCFtools 0.1.16

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://github.com/%(namelower)s/%(namelower)s/releases/download/v%(version)s/']
+sources = ['%(namelower)s-%(version)s.tar.gz']
+checksums = ['dbfc774383c106b85043daa2c42568816aa6a7b4e6abc965eeea6c47dde914e3']
+
+builddependencies = [
+    ('pkg-config', '0.29.2'),
+    ('binutils', '2.35'),
```
Comment (Micket) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb:19]: binutils is already part of the GCC toolchain (but not the GCCcore) so this is not necessary.

Also, does this use pkg-config? If so, we should add it in previous easyconfigs as well.

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb:19]: removing Binutils, pkg-config was required. I build on Ubuntu and I always find the missing pkg-config issues.

**Comment by Micket** [comment]
There seems to already be a VCFtools 0.1.16 for GCC 10.2.0
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-GCC-10.2.0.eb

I don't understand why this PR doesn't show up as a diff that just fixes the missing pkg-config builddep... this was opened after that file was merged. I can't see why it doesn't work. Can you please rebase this on develop?

**Comment by Micket** [comment]
Alternative: Can you please open a new PR where we add the missing pkg-config to all recent versions of VCFtools?

**Review by lexming — CHANGES_REQUESTED** [review]
I made a PR to this one fixing the conflicts and adding `pkg-config` to all easyconfigs of VCFtools v0.1.16. Please check https://github.com/fizwit/easybuild-easyconfigs/pull/7

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 10 out of 10 (6 easyconfigs in total)
node301.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/658f38a6f581321527d67d3f6809612b for a full test report.

**Review by lexming — APPROVED** [review]
LGTM

**Comment by lexming** [comment]
Going in, thanks @fizwit !

**Pull request merged**